src/components/Auth/index.tsx

```diff
@@ -30,46 +31,73 @@ import {
 } from '../common/EmulatorConfigProvider';
 import { EmulatorDisabled } from '../common/EmulatorDisabled';
 import { Spinner } from '../common/Spinner';
+import styles from './index.module.scss';
 import OneAccountPerEmailCard from './OneAccountPerEmailCard/OneAccountPerEmailCard';
 import PassthroughModeCard from './PassthroughModeCard/PassthroughModeCard';
 import ClearAll from './UsersCard/ClearAll';
+import TenantPicker, { useTenantFromUrl } from './UsersCard/TenantPicker';
 import UsersCard from './UsersCard/UsersCard';
 
+export const authPath = '/auth/';
+
 export const AuthRoute: React.FC = () => {
   return (
     <Suspense fallback={<AuthRouteSuspended />}>
-      <AuthWrapper />
+      <Switch>
+        <Route path={'/auth/' + `:tenantId*`}>
+          <AuthWrapper />
+        </Route>
+      </Switch>
     </Suspense>
   );
 };
 
 export default AuthRoute;
 
 export interface AuthProps {
-  updateAuthConfig(config: { auth: AuthConfig; projectId: string }): void;
+  updateAuthConfig(config: {
+    auth: AuthConfig;
+    projectId: string;
+    tenantId: string;
+  }): void;
 }
 
 export const Auth: React.FC<AuthProps> = ({ updateAuthConfig }) => {
   const auth = useEmulatorConfig('auth');
   const { projectId } = useConfig();
 
+  const [tenantId] = useTenantFromUrl();
+
   useEffect(() => {
-    updateAuthConfig({ auth, projectId });
-  }, [auth, projectId, updateAuthConfig]);
+    updateAuthConfig({ auth, projectId, tenantId });
+  }, [auth, projectId, updateAuthConfig, tenantId]);
 
   return (
-    <GridCell span={12} className="Auth">
-      <ClearAll />
-      <Elevation z="2" wrap>
-        <UsersCard />
-      </Elevation>
-      <Elevation z="2" wrap>
-        <OneAccountPerEmailCard />
-      </Elevation>
-      <Elevation z="2" wrap>
-        <PassthroughModeCard />
-      </Elevation>
-    </GridCell>
+    <Switch>
+      <Route path={authPath + `:tenantId*`}>
```

Comment:
This lets us listen to tenant in the `useTenantFromUrl` hook. The URL is the source of truth